### DISCUSSION TITLE
Light theme?

### DISCUSSION THREAD

**Discussion by louwers** [LazyVim/LazyVim#679] [General]
Is it possible to set a light theme?

Edit: Neat. It already is a light theme. I wonder how it detects which one to use.

**Comment by dpetka2001** [comment]
See this [issue](https://github.com/LazyVim/LazyVim/issues/279). Maintainer explains how it detects whether to use light or dark theme. You can either explicitly set a dark theme variation with `colorscheme` or set `vim.go.background = "dark"`.
